tests/data/metprintOutput.html

```diff
@@ -1,158 +1,158 @@
+[<span class="r10"> Info</span>] info
+[<span class="r11">   Ok</span>] success
+[<span class="r12"> Warn</span>] warning
+[<span class="r13">  Err</span>] error
+[<span class="r9"> Crit</span>] critical
+#########################
+#        Python         #
+#########################
+none
+    bold indent
+    italic indent
+HEADER:header
+DEBUG:debug
+INFO:info
+SUCCESS:success
+WARNING:warning
+ERROR:error
+CRITICAL:critical
+#########################
+#       ColorLog        #
+#########################
+none
+         <span class="r1">bold indent</span>
+         <span class="r2">italic indent</span>
+<span class="r3">header</span>
+<span class="r10">DEBUG    </span><span class="r14">debug</span>
+<span class="r11">INFO     </span><span class="r14">info</span>
+<span class="r11">SUCCESS  </span><span class="r14">success</span>
+<span class="r12">WARNING  </span><span class="r14">warning</span>
+<span class="r13">ERROR    </span><span class="r14">error</span>
+<span class="r13">CRITICAL </span><span class="r14">critical</span>
+#########################
+#       PrintTags       #
+#########################
+none
+    <span class="r1">bold indent</span>
+    <span class="r2">italic indent</span>
+<span class="r3">header</span>
+<span class="r10">[debug] debug</span>
+<span class="r10">[info] info</span>
+<span class="r11">[success] success</span>
+<span class="r15">[warn] warning</span>
+<span class="r13">[error] error</span>
+<span class="r13">[critical] critical</span>
+#########################
+#          Xa           #
+#########################
+none
+    <span class="r1">bold indent</span>
+    <span class="r2">italic indent</span>
+<span class="r16"> TITLE </span> header
+<span class="r17"> DEBUG </span> debug
+<span class="r18"> INFO </span> info
+<span class="r19"> SUCCESS </span> success
+<span class="r20"> WARNING </span> warning
+<span class="r21"> ERROR </span> error
+<span class="r22"> CRITICAL </span> critical
+#########################
+#         Lamu          #
+#########################
+none
+               <span class="r1">bold indent</span>
+               <span class="r2">italic indent</span>
+<span class="r3">header</span>
+<span class="r23">   debug</span>  : :  debug
+<span class="r10">    info</span>  : :  info
+<span class="r11"> success</span>  : :  success
+<span class="r12"> warning</span>  : :  warning
+<span class="r13">   error</span>  : :  error
+<span class="r24">critical</span>  : :  critical
+#########################
+#        Custom         #
+#########################
+none
+         <span class="r1">bold indent</span>
+         <span class="r2">italic indent</span>
+<span class="r3">header</span>
+[<span class="r4">$  Deb</span>] debug
+[<span class="r10">* Info</span>] info
+[<span class="r11">+   Ok</span>] success
+[<span class="r12">/ Warn</span>] warning
+[<span class="r13">-  Err</span>] error
+[<span class="r9">! Crit</span>] critical
+</pre>
+    </code>
+</body>
+</html>
```

Comment:
No idea why there's a diff here + why the png output is borked for tests/data/metprintOutput.html.png. I wonder if this is something to do with line endings?